lib/websync/rsyncmetadata.js

```diff
@@ -41,7 +41,7 @@ function readFile(file, pattern, callback) {
     });
 }
 
-readFile(flagsFile, /^-(\w),?\s+--([\w-]+)\s+(.+)/gm, function(matches) {
+readFile(flagsFile, /^-(\w*),?\s+--([\w-]+)\s+(.+)/gm, function(matches) {
```

Comment:
Change the * with a + like this `/^-(\w*),?\s+--([\w-]+)\s+(.+)/gm` and i'll merge the PR